lib/tool_belt/commands/branch_command.rb

```diff
@@ -0,0 +1,18 @@
+module ToolBelt
+  module Command
+    class BranchCommand < Clamp::Command
+
+      parameter "config_file", "Release configuration file"
+      option "--version", 'VERSION', "Version to build for"
+
+      def execute
```

Comment:
This command assumes you've cloned repositories already with no warning as such.